internal/cli/atlas/clusters/region_tier_autocomplete.go

```diff
@@ -69,7 +67,6 @@ func (opts *autoCompleteOpts) tierSuggestions(toComplete string) ([]string, erro
 	availableTiers := map[string]bool{}
 	for _, p := range result.Results {
 		for _, i := range p.InstanceSizes {
-			_ = i
```

Comment:
@wtrocki I saw you added these, do you remember why?

Reply:
It seems like leftover from workaround that have been required due to the SDK issue. 
We had that done in the long running PR